test/testrunner.py

```diff
@@ -143,7 +158,12 @@ def main():
             ]
         else:
             # Execute dlc_tests pytest command
-            pytest_cmds = [["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto"]]
+            pytest_cmd = ["-s", "-rA", test_path, f"--junitxml={report}", "-n=auto"]
+            if test_type == "ec2":
+                pytest_cmd += ["--reruns=1", "--reruns-delay=10"]
+                pytest_cmd[1] = "-rAR"
```

Comment:
May I know what does this last "R" do?

Reply:
I thought this would show "RERUNS" along with the summary, but it appears it does not. I can remove